packages/docz-tools/src/gatsby-theme-docz/components/Layout/Layout.module.css

```diff
@@ -0,0 +1,56 @@
+.layout {
+  min-height: 100vh;
+
+  @media (--wide-screens) {
+    display: flex;
+  }
+}
+
+.menu {
+  position: fixed;
+  top: var(--space-large);
+  z-index: 9999;
+  margin-left: var(--space-large);
+  transition: left 300ms;
+
+  @media (--wide-screens) {
+    display: none;
+  }
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 8888;
+  width: 100%;
+  height: 100vh;
+  background: var(--color-surface);
```

Comment:
Another thought, also non-blocking... we should use a consistent color as with Menu and Modal (greyBlue at 80% opacity).

I was thinking as I did the color swaps in our components that I missed the opportunity to add a `color-overlay` variable, but looking at those components we actually implement in different ways. Menu uses an [rgba value](https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Menu/Menu.css#L111) equivalent to greyBlue at 80%, while Modal uses [greyBlue and sets an opacity](https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Modal/Modal.css#L29) on the element.